### PR TITLE
Added an icon for Proxstar, it is under Systems Admin region

### PR DESCRIPTION
**Pull request by multiojuice**
Great documentation for adding these!

### CODE REVIEW

**Comment by RamZallan** [comment]
Any reason for using the chromecast logo for the icon? Other than that LGTM.

File: data/links.json [hunk 1]
```diff
@@ -212,6 +212,11 @@
         "href": "https://starrs.csh.rit.edu",
         "icon": "material:signal-wifi-4-bar-lock"
       },
+      {
+        "name": "Proxstar",
```
Comment (mbillow) [inline data/links.json]: I think the plan is to just refer to it as vm.csh/VM Management. So, just change this to “VM Management”.

File: data/links.json [hunk 2]
```diff
@@ -212,6 +212,11 @@
...
+        "href": "https://proxstar.csh.rit.edu",
+        "icon": "material:cast"
```
Comment (mbillow) [inline data/links.json]: I would suggest: “material:dvr”

**Comment by RamZallan** [comment]
:shipit: :tada:

**Pull request merged**